docs/index.md

```diff
@@ -230,6 +230,7 @@ The Snowflake provider will use the following order of precedence when determini
 ## Currently deprecated resources
 
 - [snowflake_database_old](./docs/resources/database_old)
+- [snowflake_oauth_integration](./docs/resources/oauth_integration)
```

Comment:
what are the changes comparing to this deprecated resource? they should be in the migration guide (if any)

Reply:
Added a note about oauth integration, there's pretty much no change except our new guidelines (string-bools, etc.) that I linked for guidance.